src/components/Datatable/Pagination/Pagination.stories.tsx

```diff
@@ -35,17 +35,19 @@ export const Default: Story<PaginationProps> = ({ ...args }) => {
   }, [args]);
 
   return (
-    <Pagination
-      canNextPage={canNextPage}
-      canPreviousPage={canPreviousPage}
-      isLoading={isLoading}
-      numPageButtons={args.numPageButtons}
-      pageCount={mockPageCount}
-      pageIndex={mockPageIndex}
-      onGoToPage={onGoToPage}
-      onNextPage={onNextPage}
-      onPreviousPage={onPreviousPage}
-    />
+    <div style={{ margin: '0 auto', width: '1000px' }}>
```

Comment:
why?

Reply:
this is only for storybook. it was getting too crowded in the snapshot "Go to page" took two lines

Reply:
🤦 sorry